### ISSUE TITLE
CNN_train.py 中，为什么要有 # 读测试集数据 的内容？Why should I read the test data set?

### ISSUE THREAD

**Issue by miss-peret** [wmpscc/TensorflowBaseDemo#2]
包括以下内容：
Include the following ：

def read_test_data()
x_test, y_test = read_test_data()x_batch_test, y_batch_test = tf.train.shuffle_batch([x_test, y_test], batch_size=batch_size, capacity=200,min_after_dequeue=100, num_threads=3)
 images_test, label_test = sess.run([x_batch_test, y_batch_test])
 _images_test = np.reshape(images_test, [batch_size, 49152])
 accuracy_test = sess.run(getAccuracy, feed_dict={x: _images_test, y: label_test, drop_pro: 1})
 print("test accuracy: %g" % accuracy_test)

我认为应该删除这些内容，在训练中为什么要出现测试集内容呢？
I think we should delete this content
